server_tests/test/clientTests.js

```diff
@@ -1381,19 +1276,19 @@ describe("RUST API TEST SUITE", function() {
                 done(e);
                 return;
             }
+
             //instantiate the second camera._bublTimelapse command
             setTimeout(function() {
-                testClient.bublTimelapse(sessionId, function(res) {})
-                .then( function(res) {
-                    var err = Comparison.cameraInExclusiveUseError(res);
+                return testClient.bublTimelapse(sessionId, function(res) {})
```

Comment:
Returning inside `setTimeout` is ineffective. Use `Q.delay` rather than `setTimeout`.